charts/incubator/privatebin/questions.yaml

```diff
@@ -0,0 +1,544 @@
+        - variable: main
+          label: "Main Configuration"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: name
+                label: "name"
+                description: "Set a project name to be displayed on the website"
+                schema:
+                  type: string
+                  default: ""
+              - variable: basepath
+                label: "basepath"
+                description: "The full URL, with the domain name and directories that point to the PrivateBin files. This URL is essential to allow Opengraph images to be displayed on social networks."
+                schema:
+                  type: string
+                  default: ""
+              - variable: discussion
+                label: "discussion"
+                description: "Enable or disable the discussion feature"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+              - variable: opendiscussion
+                label: "opendiscussion"
+                description: "Preselect the discussion feature"
+                schema:
+                  type: boolean
+                  default: false
+                  required: true
+              - variable: password
+                label: "password"
+                description: "Enable or disable the password feature"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+              - variable: fileupload
+                label: "fileupload"
+                description: "Enable or disable the file upload feature"
+                schema:
+                  type: boolean
+                  default: false
+                  required: true
+              - variable: burnafterreadingselected
+                label: "burnafterreadingselected"
+                description: "Preselect the burn-after-reading feature"
+                schema:
+                  type: boolean
+                  default: false
+                  required: true
+              - variable: defaultformatter
+                label: "defaultformatter"
+                description: "Which display mode to preselect by default"
+                schema:
+                  type: string
+                  default: "plaintext"
+                  required: true
+                  enum:
+                    - value: "plaintext"
+                      description: "Plain Text"
+                    - value: "syntaxhighlighting"
+                      description: "Source Code"
+                    - value: "markdown"
+                      description: "Markdown"
+              - variable: syntaxhighlightingtheme
+                label: "syntaxhighlightingtheme"
+                description: "Set a syntax highlighting theme"
+                schema:
+                  type: string
+                  default: ""
```

Comment:
if you want to also have an empty option, you can add
```yaml
- value: ""
  description: "Unset"  (or empty or whatever)
```
User will see "Unset" but the actual value will be `""`

Without that option, once the user selects a value won't be able to unset it